packages/knip/src/plugins/unocss/index.ts

```diff
@@ -0,0 +1,33 @@
+import type { EnablerPatterns } from '#p/types/config.js';
+import type { IsPluginEnabled, Plugin, ResolveConfig } from '#p/types/plugins.js';
+import { hasDependency } from '#p/util/plugin.js';
+import type { PluginConfig } from './types.js';
+
+// https://unocss.dev/guide/config-file
+
+const title = 'UnoCSS';
+
+const enablers: EnablerPatterns = ['unocss', /^@unocss\//];
```

Comment:
Let's use only `unocss` and `@unocss/cli` for better precision?

Reply:
There are other dependencies to consider, such as `@unocss/webpack`, `@unocss/postcss`

Reply:
The "enabling packages" should only be root/core modules. The rest is referenced elsewhere, in the tool configuration. The plugin returns those to Knip, and packages can then be reported as unlisted or unused.

Having more enablers is not wrong, I just think a smaller list is more precise/easier to understand for consumers. Otherwise we'd have to try and list all possible plugins/dependencies etc. in all plugins which doesn't really make sense imho.

Reply:
My previous understanding was wrong, it seems we only need `['unocss']`